tests/Paysera/WalletApi/MapperTest.php

```diff
@@ -3,14 +3,26 @@
 namespace Paysera\WalletApi;
 
 use DateTime;
+use Exception;
+use Paysera_WalletApi_Entity_ClientPermissionsToWallet;
 use Paysera_WalletApi_Entity_Location_SearchFilter;
 use Paysera_WalletApi_Entity_Transaction;
 use Paysera_WalletApi_Entity_User_Identity;
+use Paysera_WalletApi_Exception_LogicException;
 use Paysera_WalletApi_Mapper;
 use Paysera_WalletApi_Mapper_IdentityMapper;
+use Paysera_WalletApi_OAuth_Consumer;
 
 class MapperTest extends \PHPUnit_Framework_TestCase
 {
+    private $mapper;
+
+    public function setUp()
+    {
+        $this->mapper = new Paysera_WalletApi_Mapper();
+        parent::setUp();
```

Comment:
is it intended to launc parent::setUp(); at the end? in some cases it could overwrite the values you set.

Reply:
as I understand it's resolved.

Reply:
Yes, I moved it to the top.